modules/tinymce/src/core/main/ts/focus/FocusController.ts

```diff
@@ -18,6 +18,10 @@ const isEditorUIElement = (elm: Node): boolean => {
   return NodeType.isElement(elm) && FocusManager.isEditorUIElement(elm);
 };
 
+const isEditorUIElementWithoutFullscreen = (elm: Node): boolean => {
+  return NodeType.isElement(elm) && FocusManager.isEditorUIElement(elm) && !elm.className.toString().includes('tox-fullscreen');
+};
+
```

Comment:
Not sure if this is permitted to be here. In fullscreen mode, the `tox-fullscreen` class is added to the body and the document element. When the elements inside the body (not part of the editor) receives focus, the editor blur is not dispatched since the ancestor/parent of the element contains the `tox` class.

Reply:
The `tox-` is also hard coded into the core ideally the theme should export the predicate but that never happened. So I don't think it's the end of the world to hard code more things here.